.github/workflows/build-rock.yaml

```diff
@@ -60,3 +60,33 @@ jobs:
 
       - name: Run spread
         run: spread
+
+  publish-rock:
+    runs-on: ubuntu-22.04
+    needs: [build-rock]
+    # Not on pull requests; only when they're merged
+    if: github.event_name == 'push'
```

Comment:
This job is disabled on PRs but I enabled it during testing to see if it worked. Here's an example of it working: https://github.com/canonical/snapcraft-rocks/actions/runs/8474207528/job/23220346648
And here's the page for the image: https://github.com/canonical/snapcraft-rocks/pkgs/container/snapcraft

Reply:
That's a good point, I don't know why. I know that Rockcraft pulls base images/layers from ECR, but I agree that github's container registry should already have this layer. I'll look into it